PolyEngine/Engine/Src/Engine.hpp

```diff
@@ -28,32 +28,25 @@ namespace Poly
 		void Deinit();
 
 		//////////////////////////////
-		/// Registers a PhaseUpdateFunction to be executed in the Preupdate 
-		/// part of a single frame in the same order as they were passed in.
+		/// Enum used to specify the part of update to which a given phase is registered.
 		/// @see RegisterUpdatePhase()
-		/// @see RegisterPostupdatePhase()
-		/// @param phaseFunction - void function(World*)
 		//////////////////////////////
-		void RegisterPreupdatePhase(PhaseUpdateFunction phaseFunction);
+		enum class eUpdatePhaseOrder 
+		{
+			Preupdate,
```

Comment:
According to our coding style enum fields should by all capital letters with aunderscores, i.e. PRE_UPDATE, _COUNT.